libraries/AP_InertialNav/AP_InertialNav.cpp

```diff
@@ -28,16 +28,19 @@ void AP_InertialNav::update(bool high_vibes)
 
     // get the velocity relative to the local earth frame
     Vector3f velNED;
-    if (_ahrs_ekf.get_velocity_NED(velNED)) {
-        // during high vibration events use vertical position change
-        if (high_vibes) {
-            float rate_z;
-            if (_ahrs_ekf.get_vert_pos_rate_D(rate_z)) {
-                velNED.z = rate_z;
-            }
+    
+    const bool velned_ok = _ahrs_ekf.get_velocity_NED(velNED);
+    if (velned_ok) {
+        _velocity_cm = -velNED * 100; // convert from m/s in NED to cm/s in NEU
```

Comment:
Only the Z component should be inverted.

Reply:
Crap. What is wrong with me? 🤦